js/src/lib/components/InferenceWidget/widgets/TextGenerationWidget/TextGenerationWidget.svelte

```diff
@@ -128,7 +131,7 @@
 			if (output.length === 0) {
 				warning = "No text was generated";
 			} else if (model?.pipeline_tag === "text-generation") {
-				const outputWithoutInput = output.slice(text.length);
+				const outputWithoutInput = noModelLoading ? output : output.slice(text.length);
```

Comment:
not sure how does it relate to `noModelLoading` here? ^^'

Reply:
indeed, I'll look into it 👀

Reply:
Ok I think that I did this because, OF COURSE, the output returned by Inference API in a couple of instances slightly different from the one returned by Inference Endpoints. So as `noModelLoading` is only used by Inference Endpoints, I used this to identify the API currently used, and then forgot to change this. I'll see what I can do - maybe add yet another param.